src/index.js

```diff
@@ -75,20 +76,24 @@ module.exports = (apiKey, options = {}) => {
    * @return {Promise<SearchResponse>}
    */
   const request = (method = 'POST', resource = '', opts = {}) => {
-    const { timeout, ...params } = opts;
+    const { timeout, overrideIP, ...params } = opts;
     const controller = new AbortController();
     const id = typeof timeout !== 'undefined' ? setTimeout(() => controller.abort(), timeout) : undefined;
     const url = [
       hosts[currentHost],
       resource.trim().replace(/^\/+/, '')
     ].filter((s) => s).join('/');
+    const headers = {
+      'Content-Type': 'application/json; charset=UTF-8',
+      'x-placekit-api-key': apiKey,
+    };
+    if (params.countryByIP && overrideIP) {
```

Comment:
I would set it when `overrideIP` is set, with or without `countryByIP` it will improve the search relevance.